src/lib.ts

```diff
@@ -2,26 +2,37 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 
 import Rudolf from './components/Rudolf'
+import { Checker } from './typings/Checker'
 
 export type WidgetProps = {
   elementId: string
   initialPremises: string
   checker?: string
+  debug?: boolean
 }
 
+// TODO, update this to use object with named arguments. This is silly.
```

Comment:
This was a nice idea but would require rewriting the filter on the Carnap side to generate JSON.